.circleci/test-deploy.yml

```diff
@@ -12,17 +12,21 @@ workflows:
       # Make sure to include "filters: *filters" in every test job you want to run as part of your deployment.
       - cypress/run:
           filters: *filters
+          name: Standard Npm Example
           working-directory: examples/simple
-          cache-key: cache-{{ arch }}-{{ checksum "examples/simple/package.json" }}
+          cache-key: cypress-cache-{{ arch }}-{{ checksum "examples/simple/package.json" }}
```

Comment:
We may want to prepend this with a version # so it's easy to bust if need be.

Reply:
Yeah, we can always change this key to have a version or whatever name we want in the future if we need to update the cache